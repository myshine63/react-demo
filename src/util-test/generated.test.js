it('测试变化的数据', () => {
  const user = {
    name: 'tom',
    birthday: new Date(),
    age: Math.floor(Math.random() * 10)
  }
  expect(user).toMatchSnapshot({
    birthday: expect.any(Date),
    age: expect.any(Number)
  })
})