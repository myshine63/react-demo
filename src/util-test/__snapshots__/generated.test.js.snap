// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试变化的数据 1`] = `
Object {
  "age": Any<Number>,
  "birthday": Any<Date>,
  "name": "tom",
}
`;
